packages/spindle-ui/src/SegmentedControl/SegmentedControl.css

```diff
@@ -0,0 +1,125 @@
+.spui-SegmentedControl {
+  align-items: center;
+  background-color: var(--color-surface-tertiary);
+  border-radius: 42px;
```

Comment:
nits. このcomponentが、親要素（ここでは`div#root`）をはみ出る感じになっていそうです:pray:
```suggestion
  border-radius: 42px;
  box-sizing: border-box;
```
<img width="1195" alt="貼り付けた画像_6_23_23_午後1_58" src="https://github.com/openameba/spindle/assets/4669600/1fe4387f-d88e-41e0-9714-edd71bb9022e">

Reply:
ありがとうございます！！[こちら](https://github.com/openameba/spindle/pull/731#discussion_r1237914181)で修正しました（タイミング的にレビューいただいている最中に修正したのかと思います。修正遅くてすみません🙇）

※ css

Reply:
既出でしたね、失礼しました！:pray: 🙆‍♀️